ffi/src/lib.rs

```diff
@@ -1245,9 +1246,55 @@ pub unsafe extern "C" fn tx_report_entity_for_temp_id(tx_report: *mut TxReport,
     }
 }
 
-// TODO: cache
+/// Adds an attribute to the cache.
+/// `store_cache_attribute_forward` caches values for an attribute keyed by entity
```

Comment:
nit: full sentences.  I think it would be more useful to lead with the `i.e.`, preferably in the form "for fixed `a`, cache the map from `e` to `v` such that `[e a v]` is a datom in the store".  Below, you'd have "cache the map `v` to `e`."